### ISSUE TITLE
Alter output size of decode head.

### ISSUE THREAD

**Issue by lorinczszabolcs** [open-mmlab/mmsegmentation#1474]
Hi!

Any ideas about how the output size of a decode head could be changed? Should add some upconvolutions / deconvolutions in case if we need predictions at a higher resolution?

The reason is that currently there is a 4x reduction in prediction size vs. the input size, and the logits are resized using bilinear interpolation, but that might not be the best way to proceed with really small objects.

**Comment by MengzhangLI** [comment]
Hi, it is not easy to implement because (1) size of those feature maps are determined by `strides` and `dilations` in [backbone](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/models/pspnet_r50-d8.py#L11-L12) (2) `4x` is not too small, in some cases there is `32x` (3) transpose conv is not added in most decoder head such as [FCNHead](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/decode_heads/fcn_head.py). 

If you really want to use upconv/deconv, I suggest you modify certain code in decoder head. For example, in [ERFNet](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/erfnet), there is a `[UpsamplerBlock](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/backbones/erfnet.py#L148-L191)`. Although I implement it in backbone, it is ok to implement in decoder head.

Hope my answer could help you.

Best,

**Comment by lorinczszabolcs** [comment]
Thanks for the quick help!
